CHANGELOG.md

```diff
@@ -4,6 +4,189 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.18.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.18.0-rc1) - 2022-11-15
+
+* BREAKING
+  * Remove U2F support (#20141)
+* SECURITY
+  * Support Go Vulnerability Management (#21139)
```

Comment:
This should be under misc or something like that (maybe should not be even included in changelog) as it is not feature of gitea, just development process check

Reply:
sounds good. I've just removed it :)